protostar-rust/tests/e2e/running.rs

```diff
@@ -26,6 +28,56 @@ fn running_tests() {
             [PASS] test_my_test::test_my_test::test_four
             Running 1 test(s) from src/lib.cairo
             [PASS] [..]::test_fib
-            Tests: 5 passed, 1 failed
+            Tests: 6 passed, 1 failed
+        "#});
+}
+
+#[test]
+fn running_tests_with_filter() {
+    let temp = assert_fs::TempDir::new().unwrap();
+    temp.copy_from("tests/data/example_package", &["**/*"])
+        .unwrap();
+
+    let snapbox = runner();
+    let corelib = corelib_path();
+
+    snapbox
+        .current_dir(&temp)
+        .arg("two")
+        .args(["--corelib-path", corelib])
```

Comment:
Yes, it's because of temporary dir. It could also be a problem when running a binary, at least until  #2066 is addressed. And also our corelib detection is very limited atm, #2066 and #1997 should address this.